template.go

```diff
@@ -55,13 +55,16 @@ func (c *Context) Render(raw string) (out string, err error) {
 // MatchCondition checks the condition given the context
 func (c *Context) MatchCondition(condition string) (r bool) {
 	defer func() {
-		logrus.WithFields(logrus.Fields{
-			"HTTPBody":     c.HTTPBody,
-			"KafkaPayload": c.KafkaPayload,
-			"AMQPPayload":  c.AMQPPayload,
-			"condition":    condition,
-			"result":       r,
-		}).Info("running MatchCondition")
+		if r {
+			logrus.WithFields(logrus.Fields{
+				"HTTPHeader":   c.HTTPHeader,
+				"HTTPBody":     c.HTTPBody,
+				"KafkaPayload": c.KafkaPayload,
+				"AMQPPayload":  c.AMQPPayload,
+				"condition":    condition,
+				"result":       r,
+			}).Debug("running MatchCondition")
+		}
```

Comment:
how about?
```
if r => Info
else => Debug
```
in many cases, the log messages when r=false is too overwhelming.

Reply:
never mind.  on 2nd thought, i feel ok with this change, as long as easy to change log levels for mockr runtime.